cmplxfoil/CMPLXFOIL.py

```diff
@@ -267,6 +267,7 @@ def __call__(self, aeroProblem, useComplex=False, deriv=False):
         xfoil.cr09.minf1 = aeroProblem.mach  # Mach number
         xfoil.cr09.adeg = aeroProblem.alpha  # Angle of attack
         xfoil.ci04.itmax = self.getOption("maxIters")  # Iterations limit
+        xfoil.cl01.printconv = self.getOption("printConvergence")
```

Comment:
Will this throw an error in complex mode since the variable exists only in the real Fortran source? If so, you could set the option within the `if`/`else` at the beginning of `__call__`.

Reply:
This wasn't erroring out in complex mode, but setting the option in the if/else is definitely the right approach